Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1336,6 +1336,8 @@ extension BrowserViewController: MenuActionDelegate {
             switch menuAction {
             case .openNewNormalTab:
                 self.openURLInNewTab(nil, isPrivate: false, isPrivileged: true)
+                LeanplumIntegration.sharedInstance.track(event: "Open new tab")
```

Comment:
Let's be consistent with naming the events - Normal english, but all words capitalized. So 'Open New Tab' instead.